app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.hidden = False
+        self.health = health
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+    def death(self) -> None:
```

Comment:
```suggestion
    def die(self) -> None:
```

Reply:
n general, using a verb as a function name is a good practice in programming because it conveys an action.